danger-kotlin-sdk/src/main/kotlin/systems/danger/kotlin/sdk/DangerKotlinAPI.kt

```diff
@@ -10,11 +10,32 @@ interface DangerContext {
     fun fail(message: String)
     fun fail(message: String, file: String, line: Int)
     fun suggest(code: String, file: String, line: Int)
+
+    val dangerResults: DangerResults
+}
+
+interface Violation {
+    val message: String
+    val file: String?
+    val line: Int?
+}
+
+data class Meta(
+        val runtimeName: String = "Danger Kotlin",
+        val runtimeHref: String = "https://danger.systems"
+)
+
+interface DangerResults {
```

Comment:
I wouldn't expose the danger results object, but just the single arrays.

Then instead of `val dangerResults: DangerResults`:
```
    val fails: Array<Violation>
    val warnings: Array<Violation>
    val messages: Array<Violation>
    val markdowns: Array<Violation>
```

Like we do in the other Dangers.

Also `Meta` in an internal implementation detail that shouldn't be exposed.